libexec/scoop-update.ps1

```diff
@@ -136,6 +144,14 @@ function update_scoop() {
 
     shim "$currentdir\bin\scoop.ps1" $false
 
+    set_config lastupdate ([System.DateTime]::Now.ToString('o')) | Out-Null
```

Comment:
Run this `set_config` after scoop itself and buckets are all updated, just as before.

Reply:
@niheaven I find a problem about it. I move the `set_config`, but it will set `lastupdate` every time if update bucket well.

When I update next time and have configuration about `SCOOP_HOLD`, it will refresh the `lastupdate` if I did `scoop update` in $`SCOOP_HOLD` days. It may cause disable update for long time more than $`SCOOP_HOLD` days.

Reply:
Hmm, yes, so we need another config that saves the ending time. Lots of works...